assets/snippets/binary-databases.md

```diff
@@ -1,7 +1,10 @@
 Binary databases make use of the
 [MaxMind DB file format](https://maxmind.github.io/MaxMind-DB/). MaxMind
-provides official client APIs in the language listed below. The following client
-APIs are open source and licensed under the Apache License, Version 2.0.
+provides [official client APIs](/geoip/docs/databases/#official-client-apis)
+that are open source and licensed under the Apache License, Version 2.0.  We also provide a list of
```

Comment:
This is not new, but for newer things, we tend to license under Apache 2 or MIT rather than just Apache 2. This makes our libraries more compatible with GPL v2. The Ruby libraries, for instance, are licensed this way.

Reply:
Ah okay, should I mention that our open source libraries are licensed under either Apache or MIT, or could I just omit mentioning the license types?

Reply:
I personally think we could just leave it vague without specifying the particular licenses. @christophermluna do you have a preference?